src/tokens/token.ts

```diff
@@ -20,41 +15,6 @@ export interface Token {
 
 export type TokenType = "asset" | "collectible";
 
-/**
- * Load token logo from Viewblock. If no logo
- * is returned, return the original logo from
- * the contract.
- *
- * @param id Contract ID of the token
- * @param defaultLogo Default logo tx ID in the contract state
- * @param theme UI theme to match the logo with
- */
-export async function loadTokenLogo(id: string, defaultLogo?: string, theme?: DisplayTheme) {
```

Comment:
This function was only used in 1 place and you can see the TSDoc doesn't match what it does, so I guess it's an old version that was left behind. Everywhere else, `getUserAvatar` was used.